internal/config/config.go

```diff
@@ -112,8 +112,9 @@ type GeneratorConfig struct {
 // MutationConfig is the information about the GraphQL mutations.
 type MutationConfig struct {
 	// Name is the name of the GraphQL method.
-	Name               string `yaml:"name"`
-	MaxQueryFieldDepth int    `yaml:"maxQueryFieldDepth,omitempty"`
+	Name               string            `yaml:"name"`
+	MaxQueryFieldDepth int               `yaml:"max_query_field_depth,omitempty"`
+	ArgumentOverrides  map[string]string `yaml:"argument_overrides,omitempty"`
```

Comment:
This is to override the type in the call, correct?  If so, can we call it a `ArgumentTypeOverride` or something to indicate what we're overriding.  Without an example, it might be hard to guess.  Or a doc about where its used I suppose too.

Reply:
seems reasonable.  There is an example usage in the example config.